api/serverV2_test.go

```diff
@@ -0,0 +1,49 @@
+package api
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+
+	"github.com/iotexproject/iotex-core/config"
+	"github.com/iotexproject/iotex-core/testutil"
+)
+
+func TestServerV2Start(t *testing.T) {
+	require := require.New(t)
+	cfg := newConfig(t)
+	config.SetEVMNetworkID(1)
+	svr, bfIndexFile, _ := createServerV2(cfg, false)
+	defer func() {
+		testutil.CleanupPath(t, bfIndexFile)
+	}()
+
+	err := svr.Start()
+	require.NoError(err)
+
+	time.Sleep(3 * time.Second)
+	err = svr.Stop()
+	require.NoError(err)
+}
+
+func TestServerV2Error(t *testing.T) {
+	require := require.New(t)
+	cfg := newConfig(t)
+	config.SetEVMNetworkID(1)
+	svr, bfIndexFile, _ := createServerV2(cfg, false)
+	defer func() {
+		testutil.CleanupPath(t, bfIndexFile)
+	}()
+
+	svr.core.chainListener = nil
+	err := svr.Start()
+	require.Error(err)
+	require.Contains(err.Error(), "failed to add chainListener")
+
+	svr.tracer = &tracesdk.TracerProvider{}
+	err = svr.Stop()
+	require.Error(err)
+	require.Contains(err.Error(), "failed to shutdown api tracer")
```

Comment:
can combine these 2 tests into `TestServerV2Start` above? the `createServerV2` part is identical

Reply:
ok